database/migrations/2012_10_03_133633_create_base_tables.php

```diff
@@ -782,7 +782,7 @@ public function up(): void
                 $table->integer('RASoftcorePoints')->nullable()->default(0)->after('RAPoints');
 
                 // https://github.com/RetroAchievements/RAWeb/blob/master/database/20220810_000000_Add_UserAccount_SoftcorePoints_Key.sql
-                $table->index(['RASoftcorePoints', 'Untracked']);
+                $table->index(['RASoftcorePoints', 'Untracked'], 'users_points_softcore_unranked_index');
```

Comment:
`untracked`, not `unranked`.

Reply:
copied that from the User model where a todo states to rename untracked to either unranked or remove in favor of unranked_at but i'm not sold on the term `unranked` myself. that's a leftover from the v2 prototype.
adjusted in https://github.com/RetroAchievements/RAWeb/pull/1935/commits/b78ec3a435a8a5474e20a149ae209372d2b14b89